packages/hawtio/src/preferences/HomePreferences.tsx

```diff
@@ -88,7 +90,7 @@ const ResetForm: React.FunctionComponent = () => {
   return (
     <React.Fragment>
       <FormGroup label='Reset settings' fieldId='reset-form-reset'>
-        <Button variant='danger' onClick={confirmReset}>
+        <Button data-testid='reset-btn' variant='danger' onClick={confirmReset}>
```

Comment:
I guess you are able to distinguish between the `reset-btn`s with some label or the fieldId right?